src/group-configurations/group-configuration-container/GroupConfigurationContainer.scss

```diff
@@ -0,0 +1,79 @@
+.configuration-card {
+  @include pgn-box-shadow(1, "down");
+
+  background: $white;
+  border-radius: .375rem;
+  padding: 1.5rem;
```

Comment:
```suggestion
  padding: map-get($spacers, 4);
```

Reply:
i've changed, but for me it's bit of a confusing thing:)